### DISCUSSION TITLE
HELP needed, not able to run local development setup

### DISCUSSION THREAD

**Discussion by dhruvpal05** [mathesar-foundation/mathesar#3372] [Developer Q&A]
I have followed every step of the developer guide, still not able to run the server at `http://localhost:8000/`. This is the message I am getting.

![Screenshot from 2023-12-22 16-29-19](https://github.com/mathesar-foundation/mathesar/assets/91111849/c458af6a-0c84-4de9-86be-64d47107b2be)

**Comment by kgodey** [comment]
@dhruvpal05 Please ask this on [Matrix](https://wiki.mathesar.org/community/matrix/). Note that the maintainers are on holiday break until Jan 2, but a community member may be able to help before then.

**Comment by dhruvpal05** [comment]
hi, @kgodey thanks for responding, this problem was resolved.
